person_capture/gui_app.py

```
@@ -14815,12 +14942,114 @@ def _pick_trt():
+            "YOLO", "Face max", "Face detector", "Face quality", "Face visible",
+            "ReID", "Combine", "Match mode", "Disable ReID", "Require face",
+            "prefer_face", "Face ID backend", "Face fallback", "Full-frame face",
+            "Rotate", "Fast no-face", "Only best", "Lock", "Score margin",
+            "IoU", "Suppress negatives", "Neg", "drop_reid", "learn_bank",
+            "allow_faceless",
+        )
+        faceless_tokens = ("faceless",)
+        runtime_tokens = (
+            "TensorRT", "Device", "Preview", "Seek", "Overlay", "Async",
+            "JPEG", "Log interval", "Frame stride", "Min sharpness",
+            "Min seconds", "Min box", "Auto‑crop", "Border",
```

Ruff (0.15.12)

[warning] String contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)